### PR TITLE
Add typing for `self._device`

### PR DESCRIPTION
**Pull request by Necroneco**
# PR Description

Add typing for `self._device`

### CODE REVIEW

File: custom_components/midea_ac_lan/fan.py [hunk 1]
```diff
@@ -48,21 +54,9 @@ async def async_setup_entry(
 
 
 class MideaFan(MideaEntity, FanEntity):
-    def __init__(self, device: Any, entity_key: str) -> None:
+    def __init__(self, device: MideaDevice, entity_key: str) -> None:
         super().__init__(device, entity_key)
 
-    def turn_on(
```
Comment (rokam) [inline custom_components/midea_ac_lan/fan.py:54]: Why have you migrated this code?

**Review by rokam — CHANGES_REQUESTED** [review]
It looks good. Dunno why have you dupped the fan turnon method.

File: custom_components/midea_ac_lan/fan.py (hunk 1 above)
Comment (Necroneco) [reply custom_components/midea_ac_lan/fan.py:54]: I moved it to subclass, since many subclasses ovrewrite it again.

Comment (chemelli74) [reply custom_components/midea_ac_lan/fan.py:54]: We can go this way for now, but later we need to review all duplicated code and cleanup.

**Review by chemelli74 — APPROVED** [review]
Thx!

**Review by rokam — APPROVED** [review]

**Pull request merged**